mbed_tools_ci_scripts/utils/git_helpers.py

```diff
@@ -580,6 +580,20 @@ def uncommitted_changes(self) -> List[Path]:
                 line.strip().split(' ')[-1]) for line in status.splitlines()
         ]
 
+    @staticmethod
+    def _applying_modifications(destination: Path,
```

Comment:
`applying_*` is a very odd name for a method.

Reply:
any suggestion?

Reply:
`apply_*`